examples/mutators/set-label/simple/README.md

```diff
@@ -0,0 +1,36 @@
+# set-label: Simple Example
+
+The `set-label` function adds or updates labels in the `.metadata.labels` field
+and other fields that has the same meaning as a label on all resources. You can
+find more details about these fields in the help text of the function.
```

Comment:
Once we generate and host the help pages, examples should provide a link to it. Is there a tracking issue for that?

Reply:
https://github.com/GoogleContainerTools/kpt/issues/1516#issuecomment-797790042